ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -144,9 +179,8 @@ private static HttpServerOptions applyTlsKeyStore(
     final HttpServerOptions result = new HttpServerOptions(input);
 
     try {
-      final String keyStorePathname =
-          tlsConfig.getKeyStoreFile().toPath().toAbsolutePath().toString();
-      final String password = readSecretFromFile(tlsConfig.getKeyStorePasswordFile().toPath());
+      final String keyStorePathname = tlsConfig.getStoreFile().toPath().toAbsolutePath().toString();
```

Comment:
don't _think_ the absolutePath is required ... is it? (yeah, I know, it was copied)

Reply:
sorted!